From d136e7db6aa09803ff08bc2a80ea9fe30dce7f2c Mon Sep 17 00:00:00 2001
From: scx <scx.mail@gmail.com>
Date: Tue, 14 Jan 2020 23:59:55 +0100
Subject: [PATCH] Install application icon in correct directory

Move org.gnome.OCRFeeder.svg from the $(datadir)/ocrfeeder/icons/ dir
to the $(datadir)/icons/hicolor/scalable/apps/ dir.

Fixes GNOME/ocrfeeder#18
---
 resources/Makefile.am       | 5 ++++-
 resources/icons/Makefile.am | 1 -
 2 files changed, 4 insertions(+), 2 deletions(-)

diff --git a/resources/Makefile.am b/resources/Makefile.am
index faf3acf..736b3bc 100644
--- a/resources/Makefile.am
+++ b/resources/Makefile.am
@@ -1,5 +1,8 @@
 SUBDIRS = icons
 
+appicondir = $(datadir)/icons/hicolor/scalable/apps
+appicon_DATA = icons/org.gnome.OCRFeeder.svg
+
 desktopdir = $(datadir)/applications
 desktop_in_files = org.gnome.OCRFeeder.desktop.in
 desktop_DATA = $(desktop_in_files:.desktop.in=.desktop)
@@ -15,6 +18,6 @@ ui_files_DATA = spell-checker.ui menubar.ui
 
 man_MANS = ocrfeeder.1 ocrfeeder-cli.1
 
-EXTRA_DIST = $(desktop_in_files) $(appdata_in_files) $(man_MANS) $(ui_files_DATA)
+EXTRA_DIST = $(appicon_DATA) $(desktop_in_files) $(appdata_in_files) $(man_MANS) $(ui_files_DATA)
 
 CLEANFILES = $(desktop_DATA) $(appstream_DATA)
diff --git a/resources/icons/Makefile.am b/resources/icons/Makefile.am
index c6c6366..b9de08a 100644
--- a/resources/icons/Makefile.am
+++ b/resources/icons/Makefile.am
@@ -1,7 +1,6 @@
 iconsdir = $(datadir)/ocrfeeder/icons/
 icons_DATA = \
 	detect_icon.svg \
-	org.gnome.OCRFeeder.svg \
 	ocr.svg \
 	window_icon.png
 
-- 
2.24.1

